README.md

```diff
@@ -66,8 +67,8 @@ jobs:
         params:
           POOL_REPO: MY_CONCOURSE_LOCKS_REPO
           POOL_REPO_SSH_PRIVATE_KEY: ((MY_GIT_SSH_PRIVATE_KEY))
-          POOLS: MYLIST,OF,POOLS,INSIDE,THE,LOCK,REPO
-          STALE_TIMEOUT: 60
+          POOLS: POOL_1:STALE_TIMEOUT_1,POOL_2:STALE_TIMEOUT_2
+          # POOLS: POOL_1,POOL_2  # if the timeout values are ommitted, the default timeout is used (60 minutes)
```

Comment:
ommitted -> omitted.

Is there a specific reason to keep the default timeout? I would make it even more explicit and always require to specify the timeout.

Reply:
Nope, I can remove that too.